<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvod</class>
 <widget class="QDialog" name="vvod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод поверхности</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00007f;&quot;&gt;Введите уравнение поверхности&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>z=f(x;y)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>z=+/-f(x;y)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Z=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="toolTip">
      <string>Нажмите &quot;?&quot; чтобы получить справку
по вводу функций</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Введите интервал построения:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="toolTip">
      <string>Не меньше -7</string>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;x&lt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="toolTip">
      <string>Не меньше -7</string>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;y&lt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="toolTip">
      <string>Не больше 7</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Выберите цвет:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>Чёрный</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QRadioButton" name="radioButton_5">
     <property name="text">
      <string>Красный</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6" colspan="2">
    <widget class="QRadioButton" name="radioButton_7">
     <property name="text">
      <string>Фиолетовый</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="text">
      <string>Зелёный</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4" colspan="2">
    <widget class="QRadioButton" name="radioButton_6">
     <property name="text">
      <string>Синий</string>
     </property>
    </widget>
   </item>
   <item row="9" column="6" colspan="2">
    <widget class="QRadioButton" name="radioButton_8">
     <property name="text">
      <string>Жёлтый</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Ввести
 координаты</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="toolTip">
      <string>Не больше 7</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
